styles/styles.css

```diff
@@ -548,5 +548,24 @@ main .section.background-gray {
     h4 { font-size: var(--h4-font-size-desktop); }
     h5 { font-size: var(--h5-font-size-desktop); }
   }
+  
+  main .section.side-by-side .section-container {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  
+  main .section.side-by-side .section-container > div {
+      padding: 0 1rem;
+  }
+  
+  main .section.side-by-side .section-container > div:first-child {
+    flex: 0 0 58%;
```

Comment:
imho we should take this percentage from the word document to keep side-by-side more generic. wdyt?

Reply:
yes, I was also thinking same so we take this in section meta data ? or we can provide some styling OOTB, like
60-40, 50-50, 40-60 like that ?

Reply:
done. `layout-60-40`